packages/snap-preact/components/src/themes/bocachica/components/molecules/variantSelection.ts

```diff
@@ -0,0 +1,4 @@
+import type { VariantSelectionProps } from '../../../../components/Molecules/VariantSelection';
+
+// VariantSelection component props
+export const variantSelection: Partial<VariantSelectionProps> = {};
```

Comment:
and here

Reply:
This will be added once the style is completed.